tools/kvmexit.py

```diff
@@ -258,17 +193,53 @@ def valid_args_list(args):
     raise Exception("Failed to do precondition check, due to: %s." % e)
 
 try:
-    if BPF.support_raw_tracepoint_in_module():
-        # Let's firstly try raw_tracepoint_in_module
+    # Let's firstly try raw_tracepoint_module
+    #if support_raw_tracepoint_module():
+    try:
         func_entry = "RAW_TRACEPOINT_PROBE(kvm_exit)"
         get_er = "ctx->args[0]"
-    else:
-        # If raw_tp_in_module is not supported, fall back to regular tp
+    except:
+        pass
+    try:
+    # If raw_tp_module is not supported, fall back to normal tp
         func_entry = "TRACEPOINT_PROBE(kvm, kvm_exit)"
         get_er = "args->exit_reason"
+    except:
+        raise Exception("Failed to catch kvm exit reasons using tracepoint")
 except Exception as e:
     raise Exception("Failed to catch kvm exit reasons due to: %s" % e)
 
+# format output
+if is_INTEL:
+    exit_reason_header = open ("/usr/src/linux-headers-" + platform.release() + "/arch/x86/include/uapi/asm/vmx.h")
+else:
+    exit_reason_header = open ("/usr/src/linux-headers-" + platform.release() + "/arch/x86/include/uapi/asm/svm.h")
```

Comment:
I guess this depends on linux-header package is installed, what if users do not install the header package?